src/report_formats/CSV_Results/CSV_Results.xsl

```diff
@@ -232,9 +210,19 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
   </xsl:choose>
   <xsl:text>",</xsl:text>
   <xsl:value-of select="openvas:formula_quote (nvt/@oid)"/>
-  <xsl:text>,"</xsl:text>
-  <xsl:value-of select="openvas:formula_quote (nvt/cve)"/>
+
+  <xsl:text>,"</xsl:text> <!-- column "CVEs" -->
+
+  <xsl:variable name="cell_cves">
+    <xsl:for-each select="nvt/refs/ref[@type = 'cve']">
+      <xsl:value-of select="str:replace (@id, $quote, $two-quotes)"/>
+      <xsl:call-template name="newline"/>
```

Comment:
What's the reason for using a newline instead of a comma?  Seems like a change
that is not really required, at least for this PR.

I guess newline and comma are both a bit tricky for CSV because they are used
as separators.

Reply:
I changed all references to newline-separation rather than comma-separation since comma-separation appeared to make less sense in a comma-separated structure.
Also it improved readability in the spreadsheet tool.
I can change this back to use comma, though, or any other character that will not appear in references.

Reply:
I added a patch to revert to comma as separator to keep compatibility for the time being.

Reply:
I see.  Thanks for reverting.